src/code/circuit_python/setup.md

```diff
@@ -0,0 +1,86 @@
+# Setup
+
+While no specific editor or tools are _required_ for using Circuit Python,
+this documentation will make use of the recommended tools, namely the Mu
+editor and CircUp library management tools.
+
+## Installing the Circuit Python Runtime
+
+In order for your microcontroller to be able to use Circuit Python, you
+have to install board-specific firmware.
+
+1. Download the Circuit Python runtime (.uf2 file) for the Raspberry Pi
+PICO from
+[https://circuitpython.org/board/raspberry_pi_pico/](https://circuitpython.org/board/raspberry_pi_pico/).
+
+1. Hold your Gizmo's student processor BOOTSEL ("Boot Select") button down
+while connecting the USB programming cable to your computer. Once you've
+connected the cable, release the BOOTSEL button.
+
+   You should now see an external drive on your computer called "RPI-RP2".
+
+1. Copy the .uf2 file you downloaded to the "RPI-RP2" drive.
+
+   Your device will automatically reboot and reconnect as a new drive
+   called "CIRCUITPY".
+
+If your Gizmo ever gets stuck in a way that prevents you from saving new
+Circuit Python code to it or if it stops showing up as the CIRCUITPY drive
+when you connect it, you may need to re-install the runtime by following
+the above steps again.
```

Comment:
This should probably be a tip, and also needs a warning with it to make clear that this will delete any programs currently on the Gizmo, so programs should be safely stored elsewhere for safe keeping.